libraries/helpers.rb

```diff
@@ -0,0 +1,16 @@
+module Drbd
+  module Cookbook
+    module Helpers
+      def drbd_packages
+        return %w(drbd8-utils) if platform?('debian') && node['platform_version'].to_i <= 9
+        return %w(drbd90-utils kmod-drbd90) if platform_family?('rhel') && node['platform_version'].to_i >= 7
```

Comment:
Something to keep in mind (at least for folks on RHEL) is that you can install different versions of drbd from the same repo:
```
[root@dokken /]# yum search drbd
================================== N/S matched: drbd ==================================
drbd84-utils.x86_64 : Management utilities for DRBD
drbd84-utils-sysvinit.x86_64 : The SysV initscript to manage the DRBD.
drbd90-utils.x86_64 : Management utilities for DRBD
drbd90-utils-sysvinit.x86_64 : The SysV initscript to manage the DRBD.
kmod-drbd84.x86_64 : drbd84 kernel module(s)
kmod-drbd90.x86_64 : drbd90 kernel module(s)
```

Right now there's no way to configure this so perhaps it might be a good idea to make an `node['drbd']['packages']` attribute which uses this method. You'll need to add the following at the bottom so it'll work in attributes:

```ruby
Chef::Node.include ::Drbd::Cookbook::Helpers
```

Now I'm not sure if this is also available on the other platforms. Also, keep in mind you're changing the default version from 8.4 to 9.0 which is a big jump for some folks. For example, if we used this cookbook on our infra, it would break our Ganeti installs since they still require 8.4.

Reply:
I moved the packages to install to an attribute. I think the 8.4->9.0 change was accidental while I was trying to get Amazon Linux to cooperate. Would you rather I keep it or revert?

Reply:
I'd say let's keep it and make sure we mention it in the Changelog. I'm planning on doing a major bump due to this which should make people read the changelog. And since we're leaving them the option of installing the older version, that should be OK.

It probably should be a good idea to add something to the README with an example of installing the older version (at least with RHEL).